Review abstracts in Admin (#5003)
theforeman/foreman-documentation: guides/common/modules/con_managing-users-and-roles.adoc

= Managing users and roles

[role="_abstract"]

> {Project} users represent individuals who access the system and are scoped to organizations and locations.

Reviewer: ```suggestion
{Project} users represent individuals who access your {Project} and are scoped to organizations and locations.
```

I am trying to come up with an alternative for "system". What do you think?
